yb-voyager/src/srcdb/ora2pg_extract_schema.go

```diff
@@ -83,13 +54,24 @@ func ora2pgExtractSchema(source *Source, exportDir string) {
 		}
 
 		err = exportSchemaObjectCommand.Wait()
+		if outbuf.String() != "" {
+			log.Infof(`ora2pg STDOUT: "%s"`, outbuf.String())
+		}
+		if errbuf.String() != "" {
+			log.Errorf(`ora2pg STDERR in export of %s : "%s"`, exportObject, errbuf.String())
+		}
 		if err != nil {
 			utils.PrintAndLog("Error while waiting for export command exit: %v", err)
 			exportSchemaObjectCommand.Process.Kill()
 			continue
```

Comment:
Because of this `continue`, I suspect the "spinner" will continue spinning. After the line 28 (`go utils.Wait(...)`) it is a MUST to either send 1 or 0 on the `utils.WaitChannel`. Can you ensure that the channel is informed on all paths leading out of the iteration? (maybe using a proper `defer` stmt?)